### PR TITLE
Updated macOS notarization process

### PR DESCRIPTION
**Pull request by britzl**
Uses the `notarytool` instead of `altool`.

### CODE REVIEW

File: editor/scripts/notarize.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 
 
-import os, sys, shutil, re, subprocess, time
+import os, sys, shutil, re, subprocess, time, json
```
Comment (britzl) [inline editor/scripts/notarize.py:18]: The tool can return json! Yay, not more string parsing.

**Review by matgis — APPROVED** [review]
Looks good! We could maybe use the `--wait` flag and not have to poll for a result, though?

File: editor/scripts/notarize.py [hunk 2]
```diff
@@ -58,72 +58,61 @@ def _exec_command(arg_list, **kwargs):
...
+    _log('UUID for notarization request is "{}"'.format(id))
 
     while True:
         time.sleep(15)
-        _log('Checking notarization status for "{}"'.format(uuid))
-        status = get_status(uuid,notarization_username, notarization_password, notarization_itc_provider)
-        if status == "success":
+        _log('Checking notarization status for "{}"'.format(id))
+        status = get_status(id, notarization_username, notarization_password, notarization_team_id)
+        if status == "Accepted":
```
[the remark this answers is not in the document]
Comment (britzl) [reply editor/scripts/notarize.py:108]: Yes, we could, I've tested it and it works. But I want to see the average notarization times over a few days an d see if it is worth doing the polling on a Linux instance instead and then back to a macOS instance for the stapling of the receipt.

[the remark this answers is not in the document]
Comment (matgis) [reply editor/scripts/notarize.py:108]: Makes sense!

**Pull request merged**